src/components/Pool/PairsList/LoadingList/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { LoadingList } from './LoadingList.component'
+export { ListLayout } from '../PairsList.styles'
```

Comment:
it's strange to have pairlist styles here, what is the relationship with loading?

Reply:
Yes, we were defining the ListLayout twice so it's better do define it in 1 place an reuse it.